.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: rust
+
+rust:
+  - nightly
```

Comment:
I tried removing the box syntax from tests, and this builds and passes tests on stable too.